modules/rest-api/pages/rest-fts-partition-file-transfer.adoc

```diff
@@ -0,0 +1,79 @@
+= Rebalance with File Transfer
+:description: The Search-Service REST API allows new partitions to be created by file transfer.
+
+[abstract]
+{description}
+
+[#http-methods-and-uris]
+== HTTP Method and URI
+
+----
+PUT /api/managerOptions
+----
+
+[#description]
+== Description
+
+The Search Service automatically partitions its indexes across all Search nodes in the cluster, ensuring optimal distribution, following rebalance.
+
+To achieve this, in versions of Couchbase Server prior to 7.1, by default, partitions needing to be newly created were entirely _built_, on their newly assigned nodes.
+In 7.1+, by default, new partitions are created by the _transfer_ of partition files from old nodes to new nodes: this significantly enhances performance.
```

Comment:
Shall we add additional info like -> The users on 7.1+ can use this endpoint if they wish to disable this feature for some reason. (please use the proper English wordings for my proposals)

Reply:
I've added: Note that users of 7.1+ can explicitly switch the feature _off_ by means of this endpoint; in which case _partition build_ is used to establish new partitions, rather than file transfer.